website/blog/2022-08-23-Support-for-mutual-tls-in-rest-apis.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "WaveMaker support for Mutual TLS in REST API’s "
+author: Praveen Chandra
+---
+---
+
+WaveMaker supports importing REST API as a service which can be used to bind it to a widget in the UI. When the REST API is hit in the design time or runtime, a SSL connection is established between the client and server where the communication takes place in which the sensitive information is encrypted.
```

Comment:
We should mention that this is needed or works only when you proxy the imported REST API or are creating API orchestration on top of imported API